Inicio de Proyecto (ya avanzado previamente 17 commits previos)

FDASEEEM opened the pull request:
inicio de proyecto ya avanzado previamente

FDASEEEM committed:
Inicio de Proyecto (ya avanzado previamente 17 commits previos)

Cesarutoro commented:
Prueba de pull request

Cesarutoro commented on src/services/gamesApi.js:
```diff
@@ -0,0 +1,138 @@
+const API_BASE_URL = "https://hakey-api-catalogo.vercel.app/api/games";
```
Api externa para catálogo funcionando!

Commit:
Fix : se le define un contraseña concreta al login para admin panel, ya que hasta el momento cualquier clave valida + correo de admin dejaba entrar, tambien se agrega mensaje personalizado para cuando se trate de entrar con correo de admin pero la clave sea incorrecta (no es el mismo para cuando el correo no es administrador)

Cesarutoro committed:
ADD: Se agrega rama features/fixes y se prueba modificando titulo de página.

Commit:
ADD : Se agrega pagina tipo formulario para obtener crredenciales del usuario para realizar la compra en caso de no estar logeado, email, nombre, tarjeta, cvv, etc. este formulario se rutea a el boton "proceder al pago" del carrito

Cesarutoro committed:
ADD: Se agrega ruta usando path*  para capturar URL no definidas, mostrando componente NotFound personalizado y agregando botón para volver al inicio.

Commit:
ADD : Implementa el formulario de compra con validaciones y cálculo de totales, además de redirección si el carrito está vacío. Se agrega un enlace para volver al carrito y se mejora el diseño de la interfaz.

Commit:
ADD : se agrega email js para enviar correos de confirmacion al usuario

Cesarutoro committed:
FIX: Se corrige DELETE desde el panel de administrador, si la respuesta está vacía se retorna un objeto vacío.

Commit:
- Implementar fetch API para endpoints de bienvenida y confirmación
- Agregar 2 usuarios de prueba con credenciales específicas
- Enviar datos completos del pedido en confirmación de compra
- Eliminar dependencia @emailjs/browser
- Agregar documentación y scripts de prueba

Commit:
Delete : se borraro .md innnesesarios
ADD: se corrigio errores en endpoint api/usuarios permitiendo logearse con un usuario en la base de datos

Commit:
Merge branch 'features/fixes' into feature/implementacion

Commit:
se borran md innecesarios
